<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="K2A4769" name="com.genericode.Key2ActIoT" tests="4" failures="0" timestamp="07 Jan 2020 19:46:07 GMT" time="10.510" errors="1">
  <testcase name="reboot" time="7.289" classname="com.genericode.Key2ActIoT">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.Insite.qa.util.Highlighter.color(Highlighter.java:11)
at com.generic.IotHub.AgentAction(IotHub.java:58)
at com.genericode.Key2ActIoT.reboot(Key2ActIoT.java:51)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- reboot -->
  <testcase name="IOT" time="3.221" classname="com.genericode.Key2ActIoT"/>
  <testcase name="Agent" classname="com.genericode.Key2ActIoT">
    <skipped/>
  </testcase> <!-- Agent -->
  <testcase name="ManageAgent" classname="com.genericode.Key2ActIoT">
    <skipped/>
  </testcase> <!-- ManageAgent -->
</testsuite> <!-- com.genericode.Key2ActIoT -->
